skrypty/config/scripts_config.lua

```diff
@@ -458,6 +458,9 @@ function ScriptsConfig:_set_mudlet_var(options)
         if k == #var_partials then
             break
         end
+        if not parent then
```

Comment:
Zamierzona zmiana?

Reply:
Tak, z jakiegos powodu mi sie tu wywala a nie zauwazyłem żeby ta zmiana coś psuła.

Reply:
Wydaje mi się, że to dzieje się w przypadku jeżeli zmienna nie jest zadeklarowana w skryptach poprawnie. Tj. nie ma tablicy, do której by należała, więc proponowałbym to zdjąć i ew. sprawdzić w czym problem.

Reply:
Wywaliłem.